docs/Contributing.asciidoc

```diff
@@ -179,25 +184,88 @@ systemd:: systemd service definitions
 templates:: HTML templates delivered by web UI
 tools:: Development tools
 
+[[dependencies]]
+== Dependencies
+
+Here we differentiate between these types of dependencies:
+
+* Runtime: run openQA
+* Test: run the test suite of openQA itself
+* For developers: style checks, code coverage etc.
+
+You can get those dependencies in several ways:
+
+* Packaged in openSUSE and other distributions
+* Manually install from source
+* Perl modules via CPAN client
+* Use a prebuilt container
+
+We manage our dependencies via a central `dependencies.yaml` file per
+repository. We generate the `.spec` files for `rpm` as well as a `cpanfile`
+for quicker installation of all Perl modules.
+
+[[runtime-dependencies]]
+=== Runtime Dependencies
+
+For openSUSE there is `openQA` and `os-autoinst`, including several subpackages.
+That should be everything necessary to run an openQA instance.
+
+```
+zypper install openQA os-autoinst
+```
```

Comment:
That is not correct as it leaves out the worker package and hence it does not make sense to pull in os-autoinst. Instead of repeating (incorrectly) I suggest to just point to our installation instructions.

Reply:
since this is a 5min development environment setup I would prefer to not ping pong over documentations and have everything in the same section/file/page

Reply:
I changed it a bit now.

Reply:
I couldn't find this in the changes anymore. If you removed the relevant part I guess we are good and you can resolve this conversation